skymp5-server/cpp/server_guest_lib/PapyrusObjectReference.cpp

```diff
@@ -314,17 +314,18 @@ VarValue PapyrusObjectReference::SetPosition(
     // note that this change is global. So players should get new position on
     // entering area or even outside of it
     selfRefr->SetPosAndAngleSilent(
-      NiPoint3((float)static_cast<double>(arguments[0].CastToFloat()),
-               (float)static_cast<double>(arguments[1].CastToFloat()),
-               (float)static_cast<double>(arguments[2].CastToFloat())),
+      NiPoint3(float(static_cast<double>(arguments[0].CastToFloat())),
```

Comment:
This is still kinda C-style cast. Would it be ok to leave just `static_cast<double>` or `static_cast<float>` here? If double cast is needed here for some reason, then let's do a `static_cast` twice